src/utils.ts

```diff
@@ -100,7 +104,12 @@ export const packPrice = (price: string | number) => {
   if (parts.length === 1) return wholeHex.concat('0000');
```

Comment:
Hurts readability but we can force the constant `'0000'` to be a function of `MAX_DECIMAL_LENGTH` using `.concat([...Array(MAX_DECIMAL_LENGTH)].fill(0).join(''))`.

Reply:
i dont think i follow

Reply:
discussed privately with cawfree